### PR TITLE
fixup readme to match current operation

### PR DESCRIPTION
**Pull request by willcl-ark**
tiny commit to correctly apply tc-netem conditions too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +70,8 @@ Or for a custom built branch with traffic shaping rules applied:
 </node>
 ```
 
+`x`, `y` and `tc_netem` datafields are optional for all nodes.
```
Comment (pinheadmz) [inline README.md]: so are `version` and `bitcoin_config`

**Review by pinheadmz — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (willcl-ark) [reply README.md]: What happens if `version` is not present? I feel like that should be (and on a quick look looks like it may be) UB?

Comment (willcl-ark) [reply README.md]: added that `bitcoin_config` is optional in force push

Comment (pinheadmz) [reply README.md]: check out the `__init__()` in tank.py, version starts off with a default of "25.0"

Comment (pinheadmz) [reply README.md]: (but if you think it should not have a default, go ahead and switch that to none in the constructor)

File: README.md [hunk 2]
```diff
@@ -69,6 +70,8 @@ Or for a custom built branch with traffic shaping rules applied:
 </node>
 ```
 
+`x`, `y`, `bitcoin-config` and `tc_netem` datafields are optional for all nodes.
```
Comment (pinheadmz) [inline README.md]: nit `bitcoin_config`

Comment (willcl-ark) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (willcl-ark) [reply README.md]: done

**Review by pinheadmz — APPROVED** [review]
utACK (I didn't test out the netem code change) but README looks good now, merge at will, Will

**Pull request merged**